corehq/apps/data_interfaces/forms.py

```diff
@@ -367,7 +367,7 @@ def __init__(self, domain, *args, **kwargs):
             Fieldset(
                 _("Case Filters") if self.show_fieldset_title else "",
                 HTML(
-                    '<p class="help-block"><i class="fa fa-info-circle"></i> %s</p>' % self.fieldset_help_text
+                    '<p class="help-block alert alert-info"><i class="fa fa-info-circle"></i> %s</p>' % self.fieldset_help_text
```

Comment:
I think you might as well remove the `.help-block` class now that this is an alert.